packages/valory/skills/liquidity_trader_abci/payloads.py

```diff
@@ -42,7 +43,7 @@ class CheckStakingKPIMetPayload(BaseTxPayload):
     """A transaction payload for the CheckStakingKPIMetRound."""
 
     tx_submitter: str
-    kpi_met_for_the_day: Optional[bool]
+    is_staking_kpi_met: Optional[bool]
```

Comment:
Now that there are so many common attributes you can convert `CallCheckpointPayload` to:

```python
@dataclass(frozen=True)
class CallCheckpointPayload(BaseTxPayload):
    """A transaction payload for the CallCheckpointRound."""

    service_staking_state: int
    min_num_of_safe_tx_required: Optional[int]
```